script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -60,42 +60,42 @@ Arcueid's plunge back then was extraordinary.
 So I believe it's definitely possible that Vlov can be pinned down.
 However―――
 <Page17>
-"―――But, still―――"
+"―――But, still―――"//to be rechecked tl suggestion 1: replace 'still' with 'even so', 2: replace 'still' with 'even so' (note: the word used needs to be the same on both lines if a change is made.), 3: suggested new version: Is Arcueid's view of him still correct?
 ...Still, Vlov is in good condition.
-Will Arcueid's view of him still remain the same?
-If her assessment was wrong by any chance, Arcueid will be handicapped by her wound.
+Is Arcueid's assessment accurate?
+If it is wrong by any chance, she will be handicapped by her wound.
 <Page18>
 ...In the worst-case scenario, I would have to fight Vlov alone―――
 No, that's not possible.
-There was no worst-case scenario in our minds. It's not that we didn't think about it, but that there was no reason to think about it.
+There is no worst-case scenario in our minds. It's not that we didn't think about it, there is just no reason to.
 <Page19>
 That's because if Arcueid is to be defeated by Vlov, the only choice I would have left is to run away.
 <Page20>
 "Hah――――――hah."
 I reach the elevator.
 I head inside and press the button for the first floor.
 There's no more time for hesitation.
-This is no time to hesitate in the first place.
+This isn't the time to hesitate in the first place.
 I take a deep breath. I check my condition. I can move properly at the very least.
 <Page21>
-It's only the fingers holding the knife that are frozen in place, as if they were not part of my body.
+Only the fingers holding the knife are frozen in place, as if they aren't part of my body.
 <Page22>
 "――――――"
 Could this be fear?
-Am I afraid of confronting the violence... of confronting that monster?
+Am I afraid of confronting that violent... of confronting that monster?
```

Comment:
I don't think this is right. There's no na or any other grammatical construct here that would make me think this is a cut-off thought from "violent creature" or something.
> 　あの暴力に、あの怪物に向き合う事が怖いのか。

Original is better here.

Reply:
Is Shiki being afraid of the act of killing here or is he afraid of confronting Vlov?

Reply:
Not sure if it's better to have "confront" come up twice in that line or if its better to change the second "confronting" to "face"

Reply:
>あの暴力に、あの怪物に向き合う事が怖いのか。

That Violence, That Monster, Am I afraid of confronting it?

is what come to my mind..